CHANGELOG.md

```diff
@@ -7,9 +7,11 @@
 
   - Major/Minor/Patch: This is an example changelog item, usually can be the commit message.
   - Append new items to make git merging easier.
-  - refactor(test-queries): ensure `waitForElementToBeRemoved` is given a valid element
 </details>
 
+## v0.72.1
+- refactor(test-queries): ensure `waitForElementToBeRemoved` is given a valid element
```

Comment:
Uh. I broke this. I coulda swore that I tested these changes in bigmaven. But now with the release, it breaks a few tests. :/ 

I'll revert? Or pend more tests? Next week!

Reply:
I'm pending the tests and will unpend with the on-going upgrade to testing-library.

Reply:
It is also green while upgrade testing-library (besides unrelated tests due to async changes in testing-library).